src/main/flight/pid.c

```diff
@@ -146,6 +147,8 @@ void resetPidProfile(pidProfile_t *pidProfile)
         .rateAccelLimit = 0,
         .itermThrottleThreshold = 250,
         .itermAcceleratorGain = 3500,
+        .itermAcceleratorThrottleChangeMaxLimit = 0,
```

Comment:
I would like to see a default value here, I could imagine someone having a glitch or two in the rx and suddenly go from 0 to full throttle, resulting in some very high anti-gravity boosting. So even a default that matches about the fastest you can move throttle without limiting anti-gravity could be useful in protecting edge cases.

Reply:
yeah, i wanted to give some default value but decided to make sure nothing changes in the current behavior. But it would make sense values like 300.